README.md

```diff
@@ -16,30 +23,57 @@ The application uses Ruby 2.5.0. You should install it to run the application. (
 We use PostgreSQL as database. You should install it to run the app.
 If needed, you should configure the `config/database.yml` file.
 
-### Configuring the project
+> Tip
+>
+> You can use docker to easy database setup
+> ```bash
+> $ docker run --name postgres_open_jobs \
+>   -e POSTGRES_PASSWORD=postgres \
+>   -e POSTGRES_DB=opensanca_jobs_development \
+>   -e POSTGRES_USER=postgres \
+>   -p 5432:5432 -d postgres:9.6.5
+> ```
 
-Clone it
+#### Redis
 
-```
-git clone https://github.com/opensanca/opensanca_jobs.git
-cd opensanca_jobs
-```
+### Configuring the project
 
 Install the dependencies and configure the database
 
 ```
-bin/setup
+$ bin/setup
 ```
 
 Run the tests
 ```
-rspec .
+$ rspec .
 ```
 
 Start the app:
 
 ```
-bin/rails s
+$ bin/rails s
+```
+
+## Docker Development
+
+### Dependencies
+
++ [Docker](https://docs.docker.com/install/)
++ [Docker Compose](https://docs.docker.com/compose/install/)
+
+### Configuring the project
+
+Startup containers
```

Comment:
`Start the containers`

Reply:
same.. right?

Reply:
yeah. I felt that `Startup containers` was strange. But it looks ok 👍